src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -91,7 +88,7 @@ public void periodic() {
     double hoodSetpoint = 0;
     if (deploy)
       hoodSetpoint = MoPrefs.getShooterHoodSetpoint();
-    if (stow)
+    else
```

Comment:
You've already initialized hoodSetpoint to 0 on 88.  No need to set it again in the else.